portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -34,20 +38,14 @@
 @WebServlet("/comment")
 public class DataServlet extends HttpServlet {
 
-  // Response content type and redirect path.
-  private final String HTML_CONTENT_TYPE = "text/html";
-  private final String JSON_CONTENT_TYPE = "application/json";
-  private final String INDEX_PATH = "/index.html";
-  // The type of entity in database, fields in entity.
-  private final String ENTITY_TYPE = "comment";
-  private final String COMMENT_NAME = "name";
-  private final String COMMENT_TEXT = "text";
-  // The default value for undefined fields.
-  private final String DEFAULT_VAL = "";
+  // The type of entity in database and fields in entity.
+  private static final String ENTITY_TYPE = "comment";
+  private static final String COMMENT_NAME = "name";
+  private static final String COMMENT_TEXT = "text";
   // The object connected to datastore.
-  DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+  private static final DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
   // The Java to JSON converter.
-  Gson gson = new Gson();
+  public static final Gson gson = new Gson();
```

Comment:
Why public?

Reply:
oops I copied form the constant class and forgot to change signature